### DISCUSSION TITLE
Was PyRosetta removed from Anaconda?

### DISCUSSION THREAD

**Discussion by dshammers** [RosettaCommons/rosetta#281] [PyRosetta]
I'm using Miniconda3 and running 'conda install pyrosetta' returns 'package not found'. Has anyone else had this issue? Did conda change somehow?

**Comment by roccomoretti** [comment]
I don't believe that PyRosetta has been enabled in the default channels. You'll need to add the [RosettaCommons Conda channel](https://www.pyrosetta.org/downloads#h.c0px19b8kvuw) to your `.condarc` file first.
